doc/new-guidelines.md

```diff
@@ -274,15 +274,6 @@ test macros `deftest` and `is`, which are ubiquitous in the Clojure community.
   (:require [cljs-time.coerce :as time.coerce]))
 ```
 
-### Namespace aliases
```

Comment:
why not still keep it ?

Reply:
Because the linter will tell any contributor what are the expected namespace aliases during development and in CI builds. Keeping this section manually up-to-date will mean keeping it in sync with the linter rules, which seems redundant to me.